org.cloudcompare.CloudCompare.yaml

```
@@ -230,6 +230,16 @@ modules:
       - -DBUILD_tracking=OFF
       - -DBUILD_visualization=OFF
 
+  - name: Cork
+    sources:
+      - type: git
+        url: https://github.com/CloudCompare/cork.git
+        commit: c914dbabfd2e31d82da3a32c47106ae58bd8c13e
+    builddir: true
+    buildsystem: simple
+    build-commands:
+      - make -j install
```

I propose to use the `${FLATPAK_BUILDER_N_JOBS}` like the boost build does to have the ability to limit the number of concurrent builds (and keep a working desktop)

```suggestion
      - make -j${FLATPAK_BUILDER_N_JOBS} install
```